hello-strongbox-gradle/build.gradle

```diff
@@ -9,8 +13,37 @@ compileJava {
 
 description = "Example project that shows how to use gradle to build and deploy artifacts to Strongbox"
 
-ext.username = System.getProperty('credentials.username') != null ? System.getProperty('credentials.username') : mavenUser
-ext.password = System.getProperty('credentials.password') != null ? System.getProperty('credentials.password') : mavenPassword
+// publishing to GitHub Package Registry
+publishing {
+    repositories {
+        maven {
+            name = "GitHubPackages"
+            url = uri("https://maven.pkg.github.com/strongbox/strongbox-examples")
+            credentials {
+                // GITHUB_USERNAME is automatically provived by the actions/setup-java action
+                username = project.findProperty("gpr.user") ?: System.getenv("GITHUB_USERNAME")
+                // GITHUB_TOKEN is automatically provived by GitHub for every repository
+                password = project.findProperty("gpr.key") ?: System.getenv("GITHUB_TOKEN")
+            }
+        }
+    }
+    publications {
+        gpr(MavenPublication) {
```

Comment:
I'm not a Gradle super-ninja, so just for my better understanding, what does `gpr` stand for exactly?